src/components/AddBook.js

```diff
@@ -22,12 +22,20 @@ const AddBook = () => {
   };
 
   return (
-    <form onSubmit={submitBookToStore}>
-      <div className="col-auto">
-        <h3>Add Book</h3>
-        <input className="add-book" placeholder="Add Title" value={title} type="text" onChange={(e) => setTitle(e.target.value)} />
-        <input className="add-book" placeholder="Add Category" value={category} type="text" onChange={(e) => setCategory(e.target.value)} />
-        <button type="submit"> Add Book</button>
+    <form onSubmit={submitBookToStore} className="row p-3 input">
+      <div className="row input">
+        <h4 className="text-secondary">ADD NEW BOOK</h4>
+        <div className="form">
+          <div className="col-lg-4 p-2">
+            <input className="form-control" placeholder="Add Title" value={title} type="text" onChange={(e) => setTitle(e.target.value)} />
+          </div>
+          <div className="col-lg-4 p-2">
+            <input className="form-control" placeholder="Add Category" value={category} type="text" onChange={(e) => setCategory(e.target.value)} />
+          </div>
+          <div className="col-lg-4 p-2 d-grid">
+            <button type="submit" className="bg-primary text-white"> Add Book</button>
+          </div>
+        </div>
       </div>
     </form>
```

Comment:
- [ ] The form responsible for adding a new book isn't properly styled (input fields and the horizontal line before the form), Kindly update your book component to match the Zeplin design.
Zeplin design
![image](https://user-images.githubusercontent.com/33767951/141313861-73635c84-7810-4c63-af79-54a6cbe8378b.png)

- [ ] Kindly, update the text of the `title` and `category` input fields to be `Book title` and `Category` respectively.
- [ ] Kindly, update the text of the submit button to `ADD BOOK`.
- [ ] The `Category` input should be a `select` field with options of the previously given categories.